### ISSUE TITLE
[Feature request] Add full relative paths for JavaScript files by default

### ISSUE THREAD

**Issue by Kristinita** [sagold/FuzzyFilePath#37]
### Request

Add automatic detection of folder structures for JavaScript links like in the CSS links.
### Justification

For example, my project structure:

``` html
|-- example1.html
|-- folder1
|   |-- example2.html
|-- folder2
|   |-- test.js
```

If I want link to `test.js` file in `example1.html` file I need `<script src="./folder2/test.js"></script>`. But if I want link to `test.js` file in `example2.html` file, I need print in my code `<script src="../folder2/test.js"></script>`. I want, to FuzzyFilePath able to determine automatically how many points need to set the name of the file.
### Extended behavior

FuzzyFilePath for CSS:

![FuzzyFilePath for CSS](http://i.imgur.com/fnbzcUd.gif)
### Actual behavior

FuzzyFilePath for JavaScript:

![FuzzyFilePath for JavaScript](http://i.imgur.com/wWfeOaV.gif)

Thanks.

**Comment by Kristinita** [comment]
**Example 2**

Line: <https://github.com/Kristinita/Kristinita.github.io/blob/master/Giologica/Valerywork-Kiravel.html#L233>
```html
<img class="rightimg" id="Strelena" src="../anima_images/Strelena.png" alt="Стрелена" border="2px">
```
_Extended autocomplete after `src`_: `../anima_images/Strelena.png`
_Actual autocomplete after `src`_: `/anima_images/Strelena.png`


**Example 3**

Line: <https://github.com/Kristinita/Kristinita.github.io/blob/master/Giologica/Демагогия-Нестеренко.html#L91>
```html
<aside class="sidebar" style="background-image: url(../aside/Бразилия.jpg)">
```
_Extended autocomplete after `url`_: `../aside/Бразилия.jpg`
_Actual autocomplete after `url`_: `/aside/Бразилия.jpg`


Only for `css` autocompletes I have correct autocomplete path.

Thanks.

**Comment by sagold** [comment]
Hi Kristinita.

In the correct trigger configuration, try changing the `base_directory` to `false`. Currently _base_directory_  seems to break the type of insertion:

```js
{
    // html - <script src="..">
    "scope": "string\\.quote.*\\.html",
    "prefix": ["src"],
    "tagName": ["script"],
    "auto": true,
    "base_directory": false,
    "extensions": ["js"],
    "relative": true
}
```

Until i fix this, simply copy the `scopes` array to your "User Settings" and make the above mentioned changes.

Thanks for your Help.

**Comment by sagold** [comment]
Fixed with f7c70f61bc3c549ca8383bbbff9d2e475d20bc5d
